example/integration_test/shared.dart

```diff
@@ -99,8 +131,10 @@ Future<void> checkTermsAndConditionsAcceptance(
 
     await $.pumpAndSettle();
     // Tap accept or cancel.
-    if (Platform.isAndroid || Platform.isIOS) {
+    if (Platform.isAndroid) {
       await $.native.tap(Selector(text: "Got It"));
+    } else if (Platform.isIOS) {
+      await $.native.tap(Selector(text: "OK"));
```

Comment:
Is this something that was missed in version upgrade PR?

Reply:
Good question. I just noticed it now as I updated the iOS tests. Strangely enough the tests seemed to pass earlier even w/o this change and they also passed in the version upgrade PR. Maybe there was cache issue or I was running wrong version of the SDK?